# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1520027308 -3600
#      Fri Mar 02 22:48:28 2018 +0100
# Node ID d61b89d93c9c0f9f22953199207d0a3d2be6f4bb
# Parent  0f6a261ab144d7f6b2cc04aed0968122d8d05daa
fix case insensitive global search in document viewer (bug #53247)

* texinfo-parser.cc (global_search): add missing Qt:CaseInsensitive

diff --git a/libgui/src/qtinfo/texinfo-parser.cc b/libgui/src/qtinfo/texinfo-parser.cc
--- a/libgui/src/qtinfo/texinfo-parser.cc
+++ b/libgui/src/qtinfo/texinfo-parser.cc
@@ -574,17 +574,19 @@ texinfo_parser::global_search (const QSt
             continue;
 
           node_text.remove (0, node_text.indexOf ("\n"));
 
           int pos = 0;
           int founds = 0;
 
           for (; founds < words.size ()
-                 && node_text.indexOf (words.at (founds)) >= 0; founds++)
+                 && node_text.indexOf (words.at (founds), 0,Qt::CaseInsensitive)
+                    >= 0;
+               founds++)
             { }
 
           if (founds < words.size ())
             continue;
 
           founds = 0;
 
           while ((pos = re.indexIn (node_text, pos)) != -1
